### PR TITLE
[CI] Torchbench profiling

### PR DESCRIPTION
**Pull request by zzpmiracle**
related to #450 

- enalble fp16
- use nsys to profiling model benchmark

### CODE REVIEW

**Comment by zzpmiracle** [comment]
torchbench related PR: https://github.com/pai-disc/torchbenchmark/pull/1

File: .github/workflows/reusable.yml [hunk 1]
```diff
@@ -24,6 +24,9 @@ on:
       extra_envs:
         required: false
         type: string
+      extra_run_command:
```
Comment (fortianyou) [inline .github/workflows/reusable.yml:27]: can we merge `extra_run_command` and `extra_build_args` into one parameter?

File: pytorch_blade/benchmark/TorchBench/test_torch_bench.sh [hunk 2]
```diff
@@ -17,19 +17,20 @@ python3 -m pip install -q -r $script_dir/requirements.txt
 
 # setup for torchbenchmark
 benchmark_repo_dir=$HOME/.cache/torchbenchmark
-if [ -d $benchmark_repo_dir ]; then
-    rm -rf $benchmark_repo_dir
+if [ ! -d $benchmark_repo_dir ]; then
+    git clone -q https://github.com/pai-disc/torchbenchmark.git --recursive $benchmark_repo_dir
```
Comment (fortianyou) [inline pytorch_blade/benchmark/TorchBench/test_torch_bench.sh:21]: Shall we update benchmark_repo if it's already exsited?

File: .github/workflows/reusable.yml (hunk 1 above)
Comment (zzpmiracle) [reply .github/workflows/reusable.yml]: OK.

File: pytorch_blade/benchmark/TorchBench/test_torch_bench.sh (hunk 2 above)
Comment (zzpmiracle) [reply pytorch_blade/benchmark/TorchBench/test_torch_bench.sh:21]: Yes, we update after cd into the bench repo.

**Review by fortianyou — APPROVED** [review]
LGTM

**Pull request merged**